sys/console/full/syscfg.yml

```diff
@@ -93,6 +93,10 @@ syscfg.defs:
             affect device performance due to more frequent polling.
         value: 250
 
+    CONSOLE_DEFAULT_LOCK_TIMEOUT:
+        description: 'Default timeout (int ticks) for console_lock() function.'
+        value: 10
```

Comment:
10 is <=100ms on most systems. This feels a bit short to me.  Console output is slow, and I can imagine this timeout expiring when multiple tasks want to write.  Do you think something longer, such as 1 second, would be better?

Also, I would not use OS ticks are the units here.  Since OS ticks vary by platform, the default is not consistent.  I would use milliseconds here, and define a `CONSOLE_LOCK_TIMEOUT_TICKS` macro in the code.

Finally, this is a bit of a nitpick, but I don't think this setting name should contain `DEFAULT`.  It is not possible to override this setting at runtime, so it is more than just a default; it is the setting value itself!

Reply:
Changed to ms.
CONSOLE_DEFAULT_LOCK_TIMEOUT left since it specifies value used in locking console from console_write and console_out functions that do not provide timeout.